src/lens/auth.clj

```diff
@@ -1,4 +1,13 @@
-(ns lens.auth)
+(ns lens.auth
+  (:require [schema.core :as s]))
+
+(s/def UserInfo
```

Comment:
A UserInfo should not be conditional (maybe) itself. A function should return `(s/maybe UserInfo)` instead.